### DISCUSSION TITLE
Where is the setting for "Day Mode" and "Night Mode" of map style

### DISCUSSION THREAD

**Discussion by wu520609** [andreynovikov/trekarta#254] [Q&A]
Hello:
I found there is "Day Mode" and "Night Mode" map style in App.
But I can't find where is the setting? I want my map style to "Day mode" all the time.
And I found in "Night Mode",The button Zoom_in and Zoom_ out disappear.Because the icon background color is White.

**Comment by andreynovikov** [comment]
Long press Maps button (second right). It will open map menu with many customization options. Night mode is there.

**Comment by wu520609** [comment]
It is O.K Now,Thanks.
